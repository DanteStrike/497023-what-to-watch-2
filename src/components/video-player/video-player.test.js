import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

it(`Render correctly VideoPlayer component`, () => {
  const VideoPlayerComponent = renderer
    .create(
        <VideoPlayer
          isActivePlayer={false}
          src={`https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`}
          poster={`img/johnny-english.jpg`}
          isMuted={true}
        />,
        {
          createNodeMock: (element) => {
            if (element.type === `video`) {
              return {
                src: null
              };
            }
            return null;
          }
        }
    ).toJSON();

  expect(VideoPlayerComponent).toMatchSnapshot();
});
