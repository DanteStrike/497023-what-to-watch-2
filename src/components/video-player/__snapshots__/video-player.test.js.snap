// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly VideoPlayer component 1`] = `
<video
  src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
  style={
    Object {
      "display": "block",
      "height": "100%",
      "objectFit": "fill",
      "width": "100%",
    }
  }
/>
`;
